browser/resources/resource_ids

```diff
@@ -72,4 +72,8 @@
   "brave/vendor/bat-native-ads/resources/bat_ads_resources.grd": {
     "includes": [40000]
   },
+  "brave/ui/webui/resources/brave_webui_resources.grd": {
+    "includes": [41000],
```

Comment:
this is interesting; are these just arbitrarily assigned numbers? (unique of course)

Reply:
yes but they have to not conflict with chromium's existing range, and the gaps have to allow however many entries will be inside that .GRD. (e.g. in this case bat_ads_resources.grd is allowed 1000 resources)